src/usecases/disassociate-design-use-case.ts

```diff
@@ -23,11 +18,14 @@ import { jiraService } from '../infrastructure/jira';
 import { associatedFigmaDesignRepository } from '../infrastructure/repositories';
 
 export type DisassociateDesignUseCaseParams = {
-	readonly entity: {
+	readonly design: {
```

Comment:
**note:** Make explicit that association is handled only for a Design and Jira Issue.